process-engine/bpmn-studio
# 🐛 Fix timer event parsing

## Changes

Fix an issue with the internal ProcessEngine, which could cause it to be unavailable, when a TimerEvent was misconfigured.

See: 
https://github.com/process-engine/process_engine_runtime/releases/tag/v8.4.1

## Issues

PR: #1585

## How to test the changes

- Upload a ProcessModel with any kind of TimerEvent that is missing a CamundaProperty `enabled`
- Restart the Studio
- See that you'll get a list of your ProcessModels
